test/sample-test/requirements.in

```diff
@@ -0,0 +1,13 @@
+wheel
```

Comment:
Can you put a really simple README next to this file too for that instruction?

Reply:
Oh right, I just forgot to commit it. Done!